master/coords.html

```diff
@@ -3226,7 +3226,7 @@ <h3 id="InterfaceSVGTransform">Interface SVGTransform</h3>
   [SameObject] readonly attribute <a>DOMMatrix</a> <a href="coords.html#__svg__SVGTransform__matrix">matrix</a>;
   readonly attribute float <a href="coords.html#__svg__SVGTransform__angle">angle</a>;
 
-  void <a href="coords.html#__svg__SVGTransform__setMatrix">setMatrix</a>(<a>DOMMatrixReadOnly</a> matrix);
+  void <a href="coords.html#__svg__SVGTransform__setMatrix">setMatrix</a>(optional <a>DOMMatrix2DInit</a> matrix);
```

Comment:
Is the default value defined somewhere? Does a missing dictionary default to all zeros, or to an identity matrix? 2D or 3D?